index.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <div class="main-page">
+        <div>
+            <section class="navbar">
+                <header class="header">Js Documentation</header>
+
+
+                <a href="#introduction" class="active">
+                    <header class="nav-link">Introduction</header>
+                </a>
+
+
+                <a href="#what_you_should_already_know" class="active">
+                    <header class="nav-link">what you should already know</header>
+                </a>
+
+
```

Comment:
It is better and required that the header tag is used to wrap the header section and use other hx headers (h2,...h6) for the header elements. So kindly do not use so many header tags.